moped-editor/src/views/projects/projectsListView/ProjectsListViewTable.js

```diff
@@ -92,26 +85,11 @@ const ProjectsListViewTable = () => {
 
   const { hiddenColumns, setHiddenColumns } = useHiddenColumnsSettings({
     defaultHiddenColumnSettings: DEFAULT_HIDDEN_COLS,
-    storageKey: "mopedColumnConfig",
+    storageKey: "mopedProjectListColumnConfig",
```

Comment:
renamed instead of trying to use the old one, also as we potentially add more datagrid tables and storable columns, having a more granular name seems like a good idea